spec/models/scimitar/resources/mixin_spec.rb

```diff
@@ -677,18 +693,18 @@ def self.scim_timestamps_map
                         'value' => 'work@test.com'
                       }
                     ]
-                  }
+                  }.with_indifferent_case_insensitive_access()
 
                   @instance.send(
                     :from_patch_backend!,
                     nature:        'add',
                     path:          path,
-                    value:         'added_over_origina@test.com',
+                    value:         'added_over_original@test.com',
```

Comment:
Includes drive-by fix of minor but annoying typo which was replicated a lot by copy-and-paste.